gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/resources/webroot/views/registration.html

```diff
@@ -57,6 +59,28 @@
                         </div>
                     </div>
 
+
+                    <div th:if="${passwordConfig!=null}" id="message">
+                        <span>Password must respect :</span>
+
+                        <p th:if="${passwordConfig.minLength!= null}"
+                           id="length" class="invalid">Contains at least <span th:text="${passwordConfig.minLength}"/>
+                            characters</p>
+
+                        <div th:if="${passwordConfig.passwordInclude!=null}">
+                            <p th:if="${passwordConfig.passwordInclude.toString()=='NUMBERS'}"
+                               id="number" class="invalid">Contains a number</p>
+
+                            <p th:if="${passwordConfig.passwordInclude.toString()=='NUMBERS_AND_SPECIAL_CHARACTERS'}"
+                               id="specialChar" class="invalid">Contains number and special character</p>
+                        </div>
+                        <p th:if="${passwordConfig.lettersInMixedCase!= null && passwordConfig.lettersInMixedCase}"
+                           id="mixedCase" class="invalid">Contains letters in mixed case</p>
+                        <p th:if="${passwordConfig.maxConsecutiveLetters!= null}"
+                           id="maxConsecutiveLetters" class="valid">Max <span th:text="${passwordConfig.maxConsecutiveLetters}"/> consecutive letters or numbers
+                        </p>
+                    </div>
```

Comment:
Move this block under the password input.

Reply:
ok